src/priority/priority.py

```diff
@@ -339,9 +361,8 @@ def insert_stream(self,
 
         if (len(self._streams) + 1) > self._maximum_streams:
             raise TooManyStreamsError(
-                "Refusing to insert %d streams into priority tree at once" % (
-                    self._maximum_streams + 1
-                )
+                "Refusing to insert %d streams into priority tree at once"
+                % (self._maximum_streams + 1)
```

Comment:
I'd like to switch over to the new F-strings / PEP 498 to improve readability and code flow.
Would you mind including this throughout the project?

Reply:
I'll apply pyupgrade --py36-plus, and black, etc in the next PR, I'll copy in the pre-commit config from twisted